ci-connie.yml

```diff
@@ -135,9 +137,12 @@ post-processors:
     tag: "{{ user `docker_tag` }}"
     only:
     - docker
-  - type: artifice
-    files:
-    - tmp/docker-meta/.dumped
```

Comment:
The more I look at this, the more I think that the `artifice` post-processor is probably still the correct one to use.  I don't see indications in the packer docs that `artifice` is deprecated, or that we are being encouraged to switch to the `manifest` post-processor.  Instead, I suspect something may have changed with the way we are supposed to use the `shell-local` post-processor.